suite.go

```diff
@@ -13,7 +13,46 @@ type Suite struct {
 	Name   string
 	Time   string
 	Status string
-	Tests  []*Test
+	tests  []*Test
+}
+
+// AddTest adds a test case to the suite
+func (suite *Suite) AddTest(test *Test) {
```

Comment:
I'm note sure I like this change. I don't see value in hiding the data structure implementation when it's internal. I don't think the extra code is worth it and that it's very idiomatic go.

If we'd expose some data structures as an API (see #39 ) then we *might* make `Suite` and `Test` an interface.

Am I missing something?